modules/API/external/external_api.js

```diff
@@ -398,6 +399,14 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
                 }
                 break;
             }
+            case 'email-change': {
+                const user = this._participants[userID];
```

Comment:
Since we don't have a getter I don't think we need this. But it will be nice to have getEmail method. Could you please implement it.

Reply:
Sure, I'll make a getEmail function.